README.md

```diff
@@ -145,6 +145,19 @@ let line = line!["hello", format!("{name}")];
 let line = line!["bye"; 2];
 ```
 
+## Line
+
+The `text!` macro creates a `Text` that contains a sequence of lines. It is similar to the `vec!` macro.
+
+```rust
+use ratatui::prelude::*;
+use ratatui_macros::{span, line, text};
+
+let name = "world!";
+let text = text![span!(Modifier::BOLD; "hello"), line![format!("{name}")]];
```

Comment:
First the simple thing, line in a text. Then the more interesting thing, a span can also be a line.
```suggestion
let text = text![line![format!("{name}")], span!(Modifier::BOLD; "hello")];
```

Also… `"hello".bold()` is shorter than the span macro call. The only benefit would be when `span!()` works in const contexts?

Reply:
I wanted to show an example of using `span!` but I'll also add the `"hello".bold()` on the next line so users know they can do that.